### ISSUE TITLE
non-neoplastic gallbladder polyp

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11066]
**Preferred term label:**
non-neoplastic gallbladder polyp

**Synonyms**

**Definition (free text, please give PubMed ID)**
A gallbladder polyp that is not neoplastic.

**Parent term (use hpo.jax.org/app)**
gallbladder polyp

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn  -- is this really a phenotypic observation? Or is it the result of a biopsy (we would encode that using the biosample of the GA4GH Phenopacket). The scope of the HPO includes the actual observation rather then inferences. If there are phenotypic characteristics of gallbladder polyps that you would like to add please open a new issue.

**Comment by nklsbckmnn** [comment]
Gallbaldder polyp is a phenotype. Non-neoplastic gallbladder polyp is that class of phenotype typed by histology.
